src/Init/Data/Format/Basic.lean

```diff
@@ -300,11 +300,18 @@ instance : MonadPrettyFormat (StateM State) where
   startTag _         := return ()
   endTags _          := return ()
 
-/-- Pretty-print a `Format` object as a string with expected width `w`. -/
+/--
+Renders a `Format` to a string.
+* `w`: the total width
+* `indent`: the initial indentation to use for wrapped lines
+  (subsequent wrapping may increase the indentation)
+* `column`: begin the first line wrap `column` characters earlier than usual
+  (this is useful when the output String will be printed starting at `column`)
+-/
 @[export lean_format_pretty]
-def pretty (f : Format) (w : Nat := defWidth) : String :=
-  let act: StateM State Unit := prettyM f w
-  act {} |>.snd.out
+def pretty (f : Format) (w : Nat := defWidth) (indent : Nat := 0) (column := 0) : String :=
```

Comment:
As long as we're doing a big reshuffle, why not also rename `w` to `width`? The names of optional arguments are very much a part of the interface to the function, and it seems odd to have only one of them be single-letter.

Reply:
Done.